addonfactory_test_matrix_action/__init__.py

```diff
@@ -151,18 +151,20 @@ def main():
 
     supported_splunk = _generate_supported_splunk(args, path)
     pprint.pprint(f"Supported Splunk versions: {json.dumps(supported_splunk)}")
-    print(f"::set-output name=supportedSplunk::{json.dumps(supported_splunk)}")
+    with open(os.environ["GITHUB_OUTPUT"], "a") as fh:
+        print(f"supportedSplunk={json.dumps(supported_splunk)}", file=fh)
```

Comment:
Maybe use write instead of print?:
```suggestion
        fh.write(f"supportedSplunk={json.dumps(supported_splunk)}")
```

Reply:
Applies to all